app/models/pension/charges/LifetimeAllowance.scala

```diff
@@ -24,24 +24,27 @@ import utils.DecryptorInstances.bigDecimalDecryptor
 import utils.EncryptableSyntax.EncryptableOps
 import utils.EncryptorInstances.bigDecimalEncryptor
 
-case class LifetimeAllowance(amount: BigDecimal, taxPaid: BigDecimal) {
+case class LifetimeAllowance(
```

Comment:
As recommended, could we have both values default to 'None'. That way, we need only construct with LifetimeAllowance().

Also the formatting could be improved.

Reply:
Set the default values to None

Reply:
Confirmed.